drivers/volume/portworx/portworx.go

```diff
@@ -1138,7 +1188,7 @@ func (p *portworx) CreatePair(pair *stork_crd.ClusterPair) (string, error) {
 		return "", err
 	}
 
-	port := uint64(apiPort)
+	port := uint64(p.restPort)
```

Comment:
This is for the destination cluster, so it should be set to `defaultAPIPort`, it can be overwritten from the clusterpair spec below.